eole/bin/convert/convert_HF.py

```diff
@@ -240,6 +278,12 @@ def run(cls, args):
                 )
             if os.path.exists(os.path.join(args.model_dir, "tokenizer.model")):
                 tokenizer_model = os.path.join(args.model_dir, "tokenizer.model")
+            elif os.path.exists(
+                os.path.join(args.model_dir, "sentencepiece.bpe.model")
```

Comment:
At some point we should probably loop over a list of items to test. (Might be a good topic for #26.)

Reply:
yes, but at this point difficult to see if there is really a point